Adding database vulnerability assessmen scan APIs

yaakoviyun opened the pull request:
Adding database vulnerability assessmen scan APIs

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

yaakoviyun committed:
Adding database vulnerability assessmen scan APIs

Adding database vulnerability assessmen scan APIs

AutorestCI commented:
<details><summary>Swagger to SDK encountered a Subprocess error: (Azure/azure-sdk-for-go)</summary><p>

Command: ['/usr/local/bin/autorest', '/tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/readme.md', '--go', '--go-sdk-folder=/tmp/tmp9z0w7fmq/sdk', '--multiapi', '--package-version=v12.2.1-beta', '--use=@microsoft.azure/autorest.go@preview', "--user-agent='Azure-SDK-For-Go/v12.2.1-beta services'", '--verbose']
Finished with return code 1
and output:
```shell
AutoRest code generation utility [version: 2.0.4244; node: v7.10.1]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest

There is a new version of AutoRest available (2.0.4245).
 > You can install the newer version with with npm install -g autorest@latest

   Loading AutoRest core      '/tmp/.autorest/@microsoft.azure_autorest-core@2.0.4244/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4244)
   Loading AutoRest extension '@microsoft.azure/autorest.go' (preview->3.0.40)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.1.22->2.1.22)
Processing batch task - {"tag":"package-2017-10-preview"} .
FATAL: swagger-document-override/md-override-loader - FAILED
FATAL: Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 82): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 83): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 84): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 85): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 86): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 87): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 88): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 89): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 90): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 91): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 92): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 93): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 94): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 95): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 96): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 97): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
(node:42377) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 98): Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
FATAL: swagger-document/loader - FAILED
FATAL: Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
Process() cancelled due to exception : Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
Failure during batch task - {"tag":"package-2017-10-preview"} -- Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'..
  Error: Could not read 'file:///tmp/tmp9z0w7fmq/rest/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/serverDnsAliases.json'.
```

</p></details>

yaakoviyun committed:
Fixed readme file

AutorestCI commented:
Did a commit to Azure/azure-sdk-for-python:
https://github.com/Azure/azure-sdk-for-python/commit/0e285bff3405116a429680eccc3ca9fd73f999b0

AutorestCI commented:
<details><summary>Swagger to SDK encountered a Subprocess error: (Azure/azure-sdk-for-go)</summary><p>

Command: gofmt -w ./services/
Finished with return code 2
and output:
```shell
services/sql/mgmt/2014-04-01/sql/models.go:566:8: expected ';', found ':'
services/sql/mgmt/2014-04-01/sql/models.go:571:145: rune literal not terminated
services/sql/mgmt/2014-04-01/sql/models.go:573:98: expected '}', found 'for'
services/sql/mgmt/2014-04-01/sql/models.go:573:150: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:158: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:169: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:193: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:212: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:234: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:246: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:257: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:291: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1177:8: expected ';', found ':'
services/sql/mgmt/2015-05-01-preview/sql/models.go:1182:145: rune literal not terminated
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:98: expected '}', found 'for'
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:150: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:158: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:169: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:193: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:212: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:234: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:246: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:257: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:291: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1433:8: expected ';', found ':'
services/sql/mgmt/2017-03-01-preview/sql/models.go:1438:145: rune literal not terminated
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:98: expected '}', found 'for'
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:150: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:158: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:169: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:193: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:212: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:234: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:246: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:257: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:291: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1461:8: expected ';', found ':'
services/sql/mgmt/2017-10-01-preview/sql/models.go:1466:145: rune literal not terminated
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:98: expected '}', found 'for'
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:150: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:158: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:169: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:193: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:212: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:234: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:246: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:257: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:291: illegal rune literal
```

</p></details>

yaakoviyun committed:
spelling fix

AutorestCI commented:
Did a commit to Azure/azure-sdk-for-python:
https://github.com/Azure/azure-sdk-for-python/commit/3ea5e062370d6a546969fcddff7f475335b29856

AutorestCI commented:
<details><summary>Swagger to SDK encountered a Subprocess error: (Azure/azure-sdk-for-go)</summary><p>

Command: gofmt -w ./services/
Finished with return code 2
and output:
```shell
services/sql/mgmt/2014-04-01/sql/models.go:566:8: expected ';', found ':'
services/sql/mgmt/2014-04-01/sql/models.go:571:145: rune literal not terminated
services/sql/mgmt/2014-04-01/sql/models.go:573:98: expected '}', found 'for'
services/sql/mgmt/2014-04-01/sql/models.go:573:150: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:158: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:169: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:193: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:212: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:234: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:246: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:257: illegal rune literal
services/sql/mgmt/2014-04-01/sql/models.go:573:291: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1177:8: expected ';', found ':'
services/sql/mgmt/2015-05-01-preview/sql/models.go:1182:145: rune literal not terminated
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:98: expected '}', found 'for'
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:150: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:158: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:169: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:193: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:212: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:234: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:246: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:257: illegal rune literal
services/sql/mgmt/2015-05-01-preview/sql/models.go:1184:291: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1433:8: expected ';', found ':'
services/sql/mgmt/2017-03-01-preview/sql/models.go:1438:145: rune literal not terminated
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:98: expected '}', found 'for'
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:150: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:158: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:169: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:193: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:212: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:234: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:246: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:257: illegal rune literal
services/sql/mgmt/2017-03-01-preview/sql/models.go:1440:291: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1461:8: expected ';', found ':'
services/sql/mgmt/2017-10-01-preview/sql/models.go:1466:145: rune literal not terminated
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:98: expected '}', found 'for'
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:150: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:158: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:169: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:193: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:212: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:234: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:246: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:257: illegal rune literal
services/sql/mgmt/2017-10-01-preview/sql/models.go:1468:291: illegal rune literal
```

</p></details>

anuchandy commented on specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json:
```diff
@@ -0,0 +1,646 @@
+            "create",
+            "update"
+          ]
+        },
```
The storage account should be referenced using the fully qualified ARM resource id instead of passing the url and access-key. You can then enable linked access check on it reliably to ensure sufficient permission are there to perform the operation. Since this is an ARM guideline, tagging Gaurav @ravbhatnagar

yaakoviyun replied:
We already got the approval from ARM to go this way before.
The intention here is that a user will supply a SAS key (not access key) with an expiration date and privileges he chooses to supply.
This way he is more "secure" as we have a SAS key that he controls on.

anuchandy commented on specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json:
```diff
@@ -0,0 +1,646 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully executed a Vulnerability Assessment database scan."
```
Just confirming, is there any final response that contains the scan id? or it should always be retrieved using `DatabaseVulnerabilityAssessmentScans_ListByDatabase`?

yaakoviyun replied:
Currently to get the scan ID of a past scan, the user should use: DatabaseVulnerabilityAssessmentScans_ListByDatabase

If the user has a scan ID he can use DatabaseVulnerabilityAssessmentScans_Get

anuchandy commented on specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json:
```diff
@@ -0,0 +1,646 @@
+          "200": {
+            "description": "Scan result was converted successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseVulnerabilityAssessmentScansExport"
```
If scan result was already converted, does this POST call return that existing result?

yaakoviyun replied:
yes

anuchandy commented:
@yaakoviyun thanks for the response. @ravbhatnagar please sign-off since this is a new spec.

anuchandy commented:
@yaakoviyun Could you share your MS alias, wanted to start an email thread but could't locate your name/alias.

anuchandy commented:
@ravbhatnagar ping

anuchandy commented:
@ravbhatnagar can you please take a look?

ravbhatnagar commented on specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json:
```diff
@@ -0,0 +1,646 @@
+          }
+        }
+      },
+      "post": {
```
Why is a scan not created (initiated) using PUT instead of POST?

yaakoviyun replied:
Put is traditionally used to create a resource. This API is for executing a scan (doing operation on a resource), so we thought POST will be a better match.

ravbhatnagar replied:
@yaakoviyun - But this is not modeled as an action on a resource. Should it be modeled like - POST subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/vulnerabilityAssessments/{vulnerabilityAssessmentName}/initiateScan

yaakoviyun replied:
OK Will make the change in the next API-version of this API.

yaakoviyun replied:
We decided to implement it for this api version

ravbhatnagar commented:
Just one comment around the use of POST instead of PUT.

ravbhatnagar commented:
@yaakoviyun

anuchandy commented:
@yaakoviyun can you please take a look at the one comment from Gaurav?

yaakoviyun committed:
Add initiateScan to trigger scan URL

- POST
subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/vulnerabilityAssessments/{vulnerabilityAssessmentName}/initiateScan

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-go

yaakoviyun commented:
Ping.

Regards,
Yaakov

From: Anu Thomas Chandy <notifications@github.com>
Sent: Tuesday, February 27, 2018 12:05 AM
To: Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>
Cc: Yaakov Iyun <yaiyun@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-rest-api-specs] Adding database vulnerability assessmen scan APIs (#2418)


@yaakoviyun<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyaakoviyun&data=04%7C01%7Cyaiyun%40microsoft.com%7C072bd5bb2aec4b0e9adc08d57d65079b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636552795224583242%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwifQ%3D%3D%7C-1&sdata=iKKGw2bnGRxab21qf9%2Bv9Esq%2FH8zaikI%2BCD3pGp4dKA%3D&reserved=0> can you please take a look at the one comment from Gaurav?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F2418%23issuecomment-368667435&data=04%7C01%7Cyaiyun%40microsoft.com%7C072bd5bb2aec4b0e9adc08d57d65079b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636552795224583242%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwifQ%3D%3D%7C-1&sdata=17D3rgfewz7BKcG4fExPXH8Qy%2BEDZDxafJ5lzvU58lc%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALNguQg0HeteCQNOIWkbyhV_AJyEAcrjks5tYyqegaJpZM4R5YxO&data=04%7C01%7Cyaiyun%40microsoft.com%7C072bd5bb2aec4b0e9adc08d57d65079b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636552795224583242%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwifQ%3D%3D%7C-1&sdata=kUVwkpzKhAG%2F4qAhFbYC%2FyF9Kq76apRzaVyzeiqQ56s%3D&reserved=0>.

yaakoviyun commented:
@ravbhatnagar can you take a look?

ravbhatnagar commented:
Looks good!

yaakoviyun committed:
Merge branch 'master' into master

AutorestCI commented:
# Automation for azure-libraries-for-java

AutorestCI commented:
# Automation for azure-sdk-for-node

jaredmoo commented on specification/sql/resource-manager/readme.md:
````diff
@@ -468,6 +469,7 @@ batch:
   - tag: package-2014-04
 ```
 
+
````
Undo whitespace

jaredmoo commented on specification/sql/resource-manager/readme.md:
```diff
@@ -78,6 +78,7 @@ input-file:
 - Microsoft.Sql/preview/2017-03-01-preview/restorePoints.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelOperations.json
 - Microsoft.Sql/preview/2017-10-01-preview/cancelPoolOperations.json
+- Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json
```
Please add to both package-composite-v1 and -v2 sections, and package-pure-2017-10-preview

anuchandy commented:
@yaakoviyun could you address comments from @jaredmoo?

yaakoviyun committed:
Add new swagger to more packges

Add new swagger to both package-composite-v1 and -v2 sections, and
package-pure-2017-10-preview

yaakoviyun commented:
Done.

Regards,
Yaakov

From: Anu Thomas Chandy <notifications@github.com>
Sent: Friday, April 13, 2018 6:16 AM
To: Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>
Cc: Yaakov Iyun <yaiyun@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-rest-api-specs] Adding database vulnerability assessmen scan APIs (#2418)


@yaakoviyun<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyaakoviyun&data=02%7C01%7Cyaiyun%40microsoft.com%7C6aa7f8b00d3e4536cd4608d5a0ecd957%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636591861501019013&sdata=5X%2BMrgIP3%2BywmSOSUWrOImE%2FD8yd4pqGQEExRmY4yRg%3D&reserved=0> could you address comments from @jaredmoo<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjaredmoo&data=02%7C01%7Cyaiyun%40microsoft.com%7C6aa7f8b00d3e4536cd4608d5a0ecd957%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636591861501019013&sdata=kVH9MOwDJCFe%2F9up%2BP6w72nV4AjNPxbWQv9P4PwFyeU%3D&reserved=0>?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F2418%23issuecomment-381009239&data=02%7C01%7Cyaiyun%40microsoft.com%7C6aa7f8b00d3e4536cd4608d5a0ecd957%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636591861501019013&sdata=1Wj6m4HQBmyovfRjaOL5gok4RN5G7R2jZV8G%2FpM43f0%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FALNguYWRUsIvc22hpITdBUQz5Vv1FIgkks5toBhhgaJpZM4R5YxO&data=02%7C01%7Cyaiyun%40microsoft.com%7C6aa7f8b00d3e4536cd4608d5a0ecd957%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636591861501019013&sdata=sWndnLq8j5uuXxRzgrFEVcwqwOCOwDuNiXu7VGWuFCA%3D&reserved=0>.

jaredmoo reviewed (approved):
Thanks for readme.md changes, since last month there is now a new composite-v3 section. Please add there as well. I am approving now because I am on vacation and might not look at your next commit :)

yaakoviyun committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

yaakoviyun committed:
Merge branch 'master' into master

yaakoviyun committed:
Updated "package-composite-v3"

yaakoviyun commented:
ping, can you merge?

jaredmoo commented:
@yaakoviyun  There are merge conflicts again. Can you resolve? 

@ravbhatnagar it looks like you [signed off in comments](https://github.com/Azure/azure-rest-api-specs/pull/2418#issuecomment-371751834), can you approve?

yaakoviyun committed:
Merge branch 'master' into master

yaakoviyun commented:
ping...
